go/vt/discovery/replicationlag.go

```diff
@@ -0,0 +1,70 @@
+package discovery
+
+import (
+	"flag"
+	"fmt"
+)
+
+var replicationLowLag = flag.Uint("discovery_replication_low_lag", 30, "the replication lag in seconds that is considered low")
```

Comment:
Please make this a flag.Duration, and when using it, just use .Seconds. That way the command line parsing will work for 20s, 1m or any other shortcut, and will be cleaner.

(the fact that we round it to the second is an implementation detail, and we may get more granular at some point).